internal/schemas.ts

```diff
@@ -357,9 +357,13 @@ const LineType: FoxgloveEnumSchema = {
   protobufEnumName: "Type",
   description: "An enumeration indicating how input points should be interpreted to create lines",
   values: [
-    { value: 0, name: "LINE_STRIP", description: "0-1, 1-2, ..., (n-1)-n" },
-    { value: 1, name: "LINE_LOOP", description: "0-1, 1-2, ..., (n-1)-n, n-0" },
-    { value: 2, name: "LINE_LIST", description: "0-1, 2-3, 4-5, ..." },
+    {
+      value: 0,
+      name: "LINE_STRIP",
+      description: "Connected line segments: 0-1, 1-2, ..., (n-1)-n",
```

Comment:
It's somewhat confusing that the `-` in `(n-1)` means something different from the `-` in the `-n` suffix, but not sure how to edit..

Reply:
Maybe just ending with the `...`, like in the `LINE_LIST` description?